RTCStatsReport - make this the center of statistics docs (#27028)
mdn/content: files/en-us/web/api/rtcstatsreport/has/index.md

---
title: "RTCStatsReport: has() method"
short-title: has()
slug: Web/API/RTCStatsReport/has
page-type: web-api-instance-method
browser-compat: api.RTCStatsReport.has
---

{{APIRef("WebRTC")}}

The **`has()`** method of the {{domxref("RTCStatsReport")}} interface returns a boolean indicating whether a statistics report with the specified id exists.

## Syntax

```js-nolint
has(id)
```

### Parameters

- `id`

>   - : A string containing the id of the statistics dictionary to test for in the `RTCStatsReport` object.

Author: I think you're right and this is a distinction worth making. I prefer the term "statistics object" over "an object whose statistics might be present ...". Both are correct, and it's a little less of a mouthful.

```suggestion
  - : A string containing the ID of a statistics object that might be present in this `RTCStatsReport`.
```

### Return value

`true` if an element with the specified `id` exists in the `RTCStatsReport` object; otherwise `false`.

> 

Author: Note, decision not to create an example here. I could contrive one with an ID, but seems a bit pointless.
